README.md

```diff
@@ -28,85 +28,113 @@
+  * [Using Docker Desktop](https://docs.mindsdb.com/setup/self-hosted/docker-desktop). This is the fastest and recommended way to get started and have it all running.
+  * [Using Docker](https://docs.mindsdb.com/setup/self-hosted/docker). This is also simple, but gives you more flexibility on how to further customize your server.
 
-| 🎯 Use Case                 | ⚙️ Description | Python SDK | SQL |
-|---------------------------|-----------|---------| ----- |
-| [RAG](https://docs.mindsdb.com/agents/knowledge-bases?utm_medium=community&utm_source=github&utm_campaign=mindsdb%20repo)  | Comprehensive RAG that can be populated from numerous data sources | [(Python)](https://docs.mindsdb.com/sdks/python/agents_knowledge_bases) | [(SQL)](https://docs.mindsdb.com/mindsdb_sql/agents/knowledge-bases) |
-| [Agents](https://docs.mindsdb.com/agents/agent?utm_medium=community&utm_source=github&utm_campaign=mindsdb%20repo)  | Equip agents to answer questions over structured and unstructured data in MindsDB | [(Python)](https://docs.mindsdb.com/sdks/python/agents) | [(SQL)](https://docs.mindsdb.com/mindsdb_sql/agents/agent) |
-| [Automation](https://docs.mindsdb.com/mindsdb_sql/sql/create/jobs) | Automate AI-data workflows using Jobs | [(Python)](https://docs.mindsdb.com/sdks/python/create_job) | [(SQL)](https://docs.mindsdb.com/mindsdb_sql/sql/create/jobs) |
+## Connect Your Data
 
-## 💡 Example
+You can connect to hundreds of [data sources (learn more)](https://docs.mindsdb.com/integrations/data-overview). This is just an example of a Postgres database.
 
-### Connecting AI Agents to structured and unstructured data
-
-A common use case involves connecting agents to data. The following example shows how to connect an AI agent to a database so it can perform search over structured data:
-
-First we connect the datasource, in this case we connect a postgres database (you can do this via the SQL editor or SDK)
 ```sql
--- Step 1: Connect a data source to MindsDB
+
+-- Connect to demo postgres DB
 CREATE DATABASE demo_postgres_db
 WITH ENGINE = "postgres",
 PARAMETERS = {
-    "user": "demo_user",
-    "password": "demo_password",
-    "host": "samples.mindsdb.com",
-    "port": "5432",
-    "database": "demo",
-    "schema": "demo_data"
+  "user": "demo_user",
+  "password": "demo_password",
+  "host": "samples.mindsdb.com",
+  "port": "5432",
+  "database": "demo",
+  "schema": "demo_data"
 };
 
--- See some of the data in there
-SELECT * FROM demo_postgres_db.car_sales;
+```
+
+Once you've connected your data sources, you can [combine](https://docs.mindsdb.com/mindsdb_sql/sql/api/join-on), [slice it, dice it](https://docs.mindsdb.com/mindsdb_sql/sql/api/select), and [transform](https://docs.mindsdb.com/use-cases/data_enrichment/overview) it however your heart desires using good ol' standard SQL [(learn more)](https://docs.mindsdb.com/mindsdb_sql/sql/create/view). 
```

Comment:
The last link can go to SQL API (https://docs.mindsdb.com/mindsdb_sql/overview) instead of 'create view'.

Reply:
ok, that makes, sense, can you help me with that